tools/clang/test/DXC/deprecated-select-validator.hlsl

```diff
@@ -0,0 +1,14 @@
+// Test that the deprecated option, select-validator, doesn't work.
+// RUN: not %dxc -E main -T vs_6_7 -select-validator internal %s 2>&1 | FileCheck %s
+
+// CHECK: dxc failed : Unknown argument: '-select-validator'
```

Comment:
I wasn't going to suggest we might include a bespoke error message since the option was hidden all along. For some reason, this test makes me think to at least mention the possibility of a warning that the flag is ignored and a mention of the replacement environment variable. I don't mind either way.